jekyll/_cci2/next-gen-migration-guide.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: classic-docs
+title: "Migrating to next-gen Convenience Images"
+short-title: "Migrating to next-gen Convenience Images"
+description: "Migrating to next-gen Convenience Images"
+order: 30
+version:
+- Cloud
+- Server v2.x
+---
+
+* TOC
+{:toc}
+
+
+## Overview
+
+Last year (2020) CircleCI introduced the next generation (next-gen) of Convenience Images. These new images are designed to replace the legacy Convenience Images that came out during the announcement of CircleCI 2.0, about 4 years ago. The next-gen CircleCI Convenience Images are designed from the ground up for a Continuous Integration environment. They’re designed to be faster, more efficient, and more importantly, more reliable. You can learn more about all of the features on our blog post. As we begin to deprecate the legacy images, here’s information on the migration process.
+
+Moving from a legacy to next-gen image is easy and straightforward. All legacy images have a Docker namespace of “circleci” while next-gen images have a Docker namespace of “cimg”. For example, migrating from the legacy Ruby or Python image to the respective next-gen:
+
+circleci/ruby:2.3.0 -> cimg/ruby:2.3.0
+circleci/python:3.8.4 -> cimg/python:3.8.4
+
+
+## Changes
+
+### Deprecated images
```

Comment:
each MD header needs a custom kramdow id-slug so that our docs can be translated. So for example, this would be named:

```
### Depgrecated images
{: #deprecated-images }
```

Reply:
I can definitely do it, but I am curious. Jekyll already does this. The translating part can't just use that?

Reply:
@felicianotech yeah I've wondered this too 🤷‍♀️

Reply:
It's a bit of a manual process, but for I18N purposes, source docs (ie, english) have specified manual id's so that target languages (ie, just japanese for now) don't auto-generate the ids; if we don't do this, I18n target languages aren't able to inter-link to anchor headings as the translators weren't translating html link tags. 

An example:

![image](https://user-images.githubusercontent.com/12987958/130496207-bbe15c83-5cf3-4278-93f4-7545e0718aec.png)

Reply:
Gotcha. I'll add these in in my list commit.

Reply:
Done